lib/Doctrine/ORM/Query.php

```diff
@@ -326,7 +326,7 @@ protected function _doExecute()
             $this->evictEntityCacheRegion();
         }
 
-        [$sqlParams, $types] = $this->processParameterMappings($paramMappings);
+        list($sqlParams, $types) = $this->processParameterMappings($paramMappings);
```

Comment:
Was psalm complaining about this? `[$sqlParams, $types] = $this->processParameterMappings($paramMappings);` works with PHP 7.1.

Reply:
No, but it was an unintended change. PhpStorm has a bug when refactoring a file, it replace those automatically.

If it's something we want to change, I can propose another PR to change every occurence